core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/main/dot-edit-page-nav/dot-edit-page-nav.component.ts

```diff
@@ -15,6 +15,7 @@ import {
     DotTemplate,
     FeaturedFlags
 } from '@dotcms/dotcms-models';
+// eslint-disable-next-line @nx/enforce-module-boundaries
```

Comment:
Why? dot-ema is a lib

Reply:
Added an explicit comment explaining that